[RHACS] Added storage requirements for Scanner and Scanner DB (#42472)
openshift/openshift-docs: modules/scanner-requirements.adoc

= Prerequisites for installing Scanner

{product-title} includes an image vulnerability scanner called Scanner.

> This service scans images that are not already scanned by scanners integrated into image registries. Scanner also stores information about vulneribilities and scanned images in Scanner DB.

Reviewer: ```suggestion
This service scans images that are not already scanned into image registries. The Scanner stores information about vulnerabilities and scanned images in Scanner DB.
```
- The change for vulnerabilities is required, it was a misspelling. 
- I changed the sentences a little bit for easier reading. Let me know if this changes the meaning of the sentences.

[discrete]
== Memory and storage requirements for Scanner

> Scanner can scale to multiple instances. Red Hat recommends running 2 scanner instances with the following memory and storage requirements:

Reviewer 2: CPU resource also?

> Scanner can scale to multiple instances. Red Hat recommends running 2 scanner instances with the following CPU, memory, and storage requirements:

Reviewer: ```suggestion
You can scale the Scanner to multiple instances. Red Hat recommends running 2 scanner instances with the following CPU, memory, and storage requirements:
```
- Should the second instance of "scanner" be capitalized?
